pkg/operator/staticpod/certsyncpod/certsync_controller.go

```diff
@@ -79,18 +85,69 @@ func (c *CertSyncController) sync() error {
 		case apierrors.IsNotFound(err) && !cm.Optional:
 			errors = append(errors, err)
 			continue
+
 		case apierrors.IsNotFound(err) && cm.Optional:
+			// Check with the live call it is really missing
+			configMap, err = c.configmapGetter.Get(cm.Name, metav1.GetOptions{})
+			if err == nil {
+				klog.Infof("Caches are stale. They don't see configmap '%s/%s', yet it is present", configMap.Namespace, configMap.Name)
```

Comment:
isn't this something completely normal? Would at least use `V(2)`.

Reply:
possibly, I don't want to touch it for backport